### PR TITLE
Latency section

### PR DESCRIPTION
**Pull request by jayantk**
i already ran this by nagel for a legal review.

### CODE REVIEW

File: consumers/best-practices.md [hunk 1]
```diff
@@ -41,6 +41,33 @@ Querying the current price will fail if too much time has elapsed since the last
...
+This analogy suggests two simple solutions to races:
+1. Configure protocol parameters to balance the losses from smart flow against the gains from two-way flow.
+   Market makers in traditional finance implement this approach by offering a bid/ask spread and limited liquidity.
+   The limited liquidity caps the losses to smart flow, while still earning profits from the two-way flow.
+   A successful market maker tunes the spread and offered liquidity to limit adverse selection from smart traders while still interacting with two-way flow.
+2. Give the protocol a "last look" to decide which transactions to accept.
+   In traditional finance, some exchanges give market makers a chance to walk back a trade offer after a someone else has requested it.
+   Protocols can implement this technique by splitting transactions into two parts: a request and a fulfillment.
+   First, the user requests an action to be performed with the current Pyth price.
+   Then, the protocol chooses to fulfil or deny that request with a time delay, at which point it can observe any changes to the Pyth price.
```
Comment (ali-bahjati) [inline consumers/best-practices.md:67]: I found this one a bit difficult to understand. Maybe worth mentioning decision executing with the new price.

**Review by ali-bahjati — APPROVED** [review]
Seems very professional :D I think it's good and people who understand tradfi will get it. I would like it to be simpler but I am not sure whether it's possible.

File: consumers/best-practices.md [hunk 2]
```diff
@@ -41,6 +41,33 @@ Querying the current price will fail if too much time has elapsed since the last
 The SDKs expose this failure condition in an idiomatic way: for example, the Rust SDK may return `None`, and our Solidity SDK may revert the transaction.
 The SDK provides a sane default for the staleness threshold, but users may configure it to suit their use case.
 
+# Latency
```
Comment (tompntn) [inline consumers/best-practices.md:44]: I wonder if this should go underneath the Confidence Intervals section? I think latency is a concern for a subset of integrators, whereas all integrators need to be aware of CIs.

File: consumers/best-practices.md [hunk 3]
```diff
@@ -41,6 +41,33 @@ Querying the current price will fail if too much time has elapsed since the last
...
+In this threat model, protocol designers should avoid situations where a Pyth price update must race against an adversary's transaction.
+The adversary has a head start in these races.
+Furthermore, sophisticated adversaries can highly optimize their network latencies, or pay miners for priority blockspace, such that they are virtually guaranteed to win these races.
+
+This situation is analogous to the one faced by market makers in the traditional finance.
+These market makers place resting orders on exchanges with the hope of earning the bid/ask spread.
+When the “true price” moves, these market makers get picked off by adverse “smart flow” that is faster than they are.
+The smart flow is balanced by two-way flow, that is, people wanting to trade for other reasons besides a price change.
+
+This analogy suggests two simple solutions to races:
```
Comment (tompntn) [inline consumers/best-practices.md:58]: (Not for this PR) but I think diagrams illustrating the problem and the two proposed solutions would be helpful here. They will probably make it more likely that integrators actually read this.

**Review by tompntn — APPROVED** [review]
Nice!

**Pull request merged**